tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
+
+[ -z "${KUBECONFIG}" ] && echo "KUBECONFIG is not set" && exit 1
+
+rm -rf _out
+hack/dockerized hack/build-func-tests.sh
```

Comment:
Never recommend to use targets which are not Makefile targets. We change them without notice and we don't grep the docs for that.

Reply:
Are you suggesting that this script is added as a target to the Makefile?